### PR TITLE
CHANGE(grafana): Allow use of OpenIO package mirror

### PR DESCRIPTION
**Pull request by vincent-legoll**
##### SUMMARY
This will enable the installation of grafana package from our own
mirror

 ##### ISSUE TYPE
- Feature Pull Request

 ##### SCOPE (skeleton only)
- SDS

 ##### IMPACT
N/A

 ##### ADDITIONAL INFORMATION

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ grafana_repo_release: stable
 grafana_service_name: grafana-server
 grafana_service_enabled: true
 grafana_version: 5.0.0
+grafana_upstream_pkg_repository: true
```
Comment (cdelgehier) [inline defaults/main.yml]: pkg ? package

**Review by cdelgehier — CHANGES_REQUESTED** [review]
why not set it to false? I already see your next PR on the skeleton

File: defaults/main.yml (hunk 1 above)
Comment (vincent-legoll) [reply defaults/main.yml]: because I'd like to test it manually first

**Comment by vincent-legoll** [comment]
because I'd rather test it manually first.
but I'll make the pkg => package change

**Pull request closed without merging**